core/interception.js

```diff
@@ -1,19 +1,45 @@
 // global variables
 var readConfirmationsHookEnabled = true;
 var presenceUpdatesHookEnabled = true;
+var saveDeletedMsgsHook = false
 var safetyDelay = 0;
 var WAdebugMode = false;
 var isInitializing = true;
 var exceptionsList = [];
 var blinkingChats = {};
 var chats = {};
 var blockedChats = {};
+var deletedDB = indexedDB.open("deletedMsgs", 1)
+
+deletedDB.onupgradeneeded = function (e)
```

Comment:
Maybe move the `deletedDB` related functions to more bottom in the file where there are other message-keeping functions.

Reply:
moved to the bottom :)